app/models/user.rb

```diff
@@ -145,23 +145,43 @@ def sync_profile
   devise :invitable, :omniauthable, :registerable, :database_authenticatable, :confirmable, :rememberable,
          :recoverable, :lockable
 
+  validates :articles_count, presence: true
```

Comment:
there's a bunch of sorting lines going on here, in addition to the new validators